var/spack/repos/builtin/packages/glab/package.py

```diff
@@ -34,20 +36,38 @@ class Glab(GoPackage):
     version("1.21.1", sha256="8bb35c5cf6b011ff14d1eaa9ab70ec052d296978792984250e9063b006ee4d50")
     version("1.20.0", sha256="6beb0186fa50d0dea3b05fcfe6e4bc1f9be0c07aa5fa15b37ca2047b16980412")
 
-    depends_on("go@1.13:", type="build")
-    depends_on("go@1.17:", type="build", when="@1.22:")
-    depends_on("go@1.18:", type="build", when="@1.23:")
-    depends_on("go@1.19:", type="build", when="@1.35:")
-    depends_on("go@1.21:", type="build", when="@1.37:")
-    depends_on("go@1.22.3:", type="build", when="@1.41:")
-    depends_on("go@1.22.4:", type="build", when="@1.42:")
-    depends_on("go@1.22.5:", type="build", when="@1.44:")
-    depends_on("go@1.23:", type="build", when="@1.46:")
-    depends_on("go@1.23.2:", type="build", when="@1.48:")
-    depends_on("go@1.23.4:", type="build", when="@1.52:")
+    with default_args(type="build"):
+        depends_on("go@1.24.1:", when="@1.54:")
+        depends_on("go@1.23.4:", when="@1.52:")
+        depends_on("go@1.23.2:", when="@1.48:")
+        depends_on("go@1.23:", when="@1.46:")
+        depends_on("go@1.22.5:", when="@1.44:")
+        depends_on("go@1.22.4:", when="@1.42:")
+        depends_on("go@1.22.3:", when="@1.41:")
+        depends_on("go@1.21:", when="@1.37:")
+        depends_on("go@1.19:", when="@1.35:")
+        depends_on("go@1.18:", when="@1.23:")
+        depends_on("go@1.17:", when="@1.22:")
+        depends_on("go@1.13:")
```

Comment:
```suggestion
    with default_args(type="build"):
        depends_on("go@1.24.1:", when="@1.54:")
        depends_on("go@1.23.4:", when="@1.52:")
        depends_on("go@1.23.2:", when="@1.48:")
        depends_on("go@1.23.0:", when="@1.46:")
        depends_on("go@1.22.5:", when="@1.44:")
        depends_on("go@1.22.4:", when="@1.42:")
        depends_on("go@1.22.3:", when="@1.41:")
        depends_on("go@1.21.0:", when="@1.37:")
        depends_on("go@1.19.0:", when="@1.35:")
        depends_on("go@1.18.0:", when="@1.23:")
        depends_on("go@1.17.0:", when="@1.22:")
        depends_on("go@1.13.0:")
```

Small OCD suggestion to align all of the versions to make them look a bit neater. Otherwise this looks good to me.

Reply:
Glad I am not the only one suffering from CDO 😄